windows-server-containers-preview/azuredeploy.json

```diff
@@ -0,0 +1,271 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+
+     "VMName": {
+      "type": "string",
+      "metadata": {
+        "description": "This name will also be used for the network security group, storage, storage container, virtual network, network card, subnet and public IP address name."
+      }
+    },
+
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+
+    "dnsNameForPublicIP": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+      }
+    }
+  },
+
+  "variables": {
+    "location": "West US",
+    "imagePublisher": "MicrosoftWindowsServer",
+    "imageOffer": "WindowsServer",
+    "addressPrefix": "10.0.0.0/16",
+    "subnetPrefix": "10.0.0.0/24",
+    "storageAccountType": "Standard_LRS",
+    "publicIPAddressType": "Dynamic",
+    "vmSize": "Standard_A0",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',parameters('VMName'))]",
+    "subnetRef": "[concat(variables('vnetID'),'/subnets/',parameters('VMName'))]",
+    "windowsOSVersion": "2016-Technical-Preview-3-with-Containers",
+    "globalnamelower":  "[tolower(parameters('VMName'))]"
+  },
+  "resources": [
+
+    {
+
+      "type": "Microsoft.Network/networkSecurityGroups",
+      "name": "[parameters('VMName')]",
+      "apiVersion": "2015-05-01-preview",
```

Comment:
It's a good idea to make apiVersion a variable so it's easier to migrate to newer API headers